creare: tables blueprint

FlexiblePrintedCircuits opened the pull request:
# change point
Table blueprints have been created.

# issue
#35

halsk commented:
@FlexiblePrintedCircuits Thanks! It looks awesome. 
I have one question. How do we use the Users table?

FlexiblePrintedCircuits commented:
@halsk 
When logically deleting users, I thought we should separate the users table from the profile table, because if we physically delete only the users table, the login IDs will not be missing.

However, there was an error in the table design, which has just been corrected.
Added a deletion_date column and changed the primary key type.

It is also meant to prepare you for managing your own user tokens in the future.

halsk reviewed (approved):
Thnaks, LGTM so far.
